app/services/github_handler.rb

```diff
@@ -45,6 +45,14 @@ def handle_review_request
     pr.review_requests.create!(data: payload, owner: owner, reviewer: reviewer)
   end
 
+  def handle_review_removal
+    pr = PullRequest.find_by!(github_id: payload['pull_request']['id'])
+    pr.transaction do
+      pr.review_requests = []
```

Comment:
why is this needed? Isn't `dependent: :destroy` here https://github.com/rootstrap/rs-code-review-metrics/blob/master/app/models/pull_request.rb#L26 going to take care of this already?

Reply:
You are right! Committing the changes